kinesis-video-pic/src/client/src/Stream.c

```diff
@@ -1499,14 +1499,6 @@ STATUS getStreamData(PKinesisVideoStream pKinesisVideoStream, UPLOAD_HANDLE uplo
         deleteStreamUploadInfo(pKinesisVideoStream, pUploadHandleInfo);
         pUploadHandleInfo = NULL;
         pUploadHandleInfo = getStreamUploadInfoWithState(pKinesisVideoStream, UPLOAD_HANDLE_STATE_ACTIVE);
-
```

Comment:
Just realized that I shouldn't remove this clause. Putting it back and add comment. It's self-priming in case a upload handle is terminated and there is no other active upload handle to take over.

Reply:
Yeah, that's what I deduced. Could you remove LN 1503 then?

Reply:
done

Reply:
done